HHS/Head-Start-TTADP
# [TTAHUB-1571] Add "report text" and "group" filters to goal dashboard

## Description of change
Add two new filters to the goal dashboard (Groups & Report text). Alter the existing AR filters to return Goals instead.

## How to test
Visit the goal dashboard, confirm that the filters are there, in alpha order, and return sensible results.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1571


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
